yarn.lock

```diff
@@ -1261,11 +1313,75 @@
   dependencies:
     defer-to-connect "^1.0.1"
 
+"@truffle/blockchain-utils@^0.0.15":
```

Comment:
It seems we are pulling in several oldish truffle packages. Can we avoid this somehow? It's not critical though, since this is only for development.

Reply:
These are actually not old, and are dependencies of `@truffle/contract`, which `test-environment` depends on.